bike-assistant-app/src/app/tab1/tab1.page.ts

```diff
@@ -70,6 +90,8 @@ export class Tab1Page  {
   }
 
   startTracking() {
```

Comment:
a co z zakończeniem trackowania ?

Reply:
Pomiar czasu działa od momentu ustawienia this.isTracking na  true do momentu zmiany na false. Czyli przy takim podejściu w nie musiałem modyfikować zakończenia.